<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMakeProjectManager::Internal::XMakeBuildSettingsWidget</class>
 <widget class="QWidget" name="XMakeProjectManager::Internal::XMakeBuildSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="configure_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="wipe_button">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Wipe</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Utils::DetailsWidget" name="container" native="true">
    <widget class="QWidget" name="details" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
        <item>
        <widget class="QWidget" name="build_dir_widget" native="true"/>
       </item>
      <item>
         <widget class="Utils::FancyLineEdit" name="parameters_line_edit">
                <property name="placeholderText">
          <string>Parameters</string>
         </property>
         </widget>
        </item>
      <item>
       <widget class="Utils::FancyLineEdit" name="options_filter_line_edit">
                <property name="placeholderText">
          <string>Filter</string>
         </property>
         </widget>
      </item>
      <item>
       <widget class="Utils::TreeView" name="options_tree_view">
       <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
      </item>
     </layout>
    </widget>
 </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::DetailsWidget</class>
   <extends>QWidget</extends>
   <header>utils/detailswidget.h</header>
   <container>1</container>
  </customwidget>
    <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>utils/fancylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::TreeView</class>
   <extends>QTreeView</extends>
   <header>utils/itemviews.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
